examples/openfin/app.json

```diff
@@ -6,6 +6,10 @@
         "url": "http://localhost:8000/index.html",
         "uuid": "0td3jltz4vswz5mi",
         "autoShow": true,
+        "defaultWidth": 850,
```

Comment:
Bootstrap dynamic grid columns are a bit annoying for larger displays with regards to wrapping so make main window wider.